src/target.cpp

```diff
@@ -1093,22 +1093,8 @@ const char *target_dynamic_linker(const ZigTarget *target) {
         case OsWatchOS:
         case OsMacOSX:
         case OsUefi:
-            return nullptr;
-
         case OsWindows:
-            switch (target->abi) {
```

Comment:
I checked and I can confirm that there is no such thing as a dynamic linker for mingw. This diff is :+1: